boards/arm/nrf9160dk_nrf52840/doc/index.rst

```diff
@@ -123,37 +123,116 @@ Board controller firmware
-+-----------------+----------------------------------+
-| MCU Interface 0 | Arduino pin headers              |
-+-----------------+----------------------------------+
-| MCU Interface 1 | Trace interface                  |
-+-----------------+----------------------------------+
-| MCU Interface 2 | COEX interface                   |
-+-----------------+----------------------------------+
-
-It is possible to configure the behavior of the board controller firmware by
-using Kconfig and editing its options under "Board options".
-
++--------------------------------+----------------------------------+
+| nRF9160 pins                   | Routed to                        |
++================================+==================================+
+| P0.26, P0.27, P0.28, and P0.29 | VCOM0                            |
++--------------------------------+----------------------------------+
+| P0.01, P0.00, P0.15, and P0.14 | VCOM2                            |
++--------------------------------+----------------------------------+
+| P0.02                          | LED1                             |
++--------------------------------+----------------------------------+
+| P0.03                          | LED2                             |
++--------------------------------+----------------------------------+
+| P0.04                          | LED3                             |
++--------------------------------+----------------------------------+
+| P0.05                          | LED4                             |
++--------------------------------+----------------------------------+
+| P0.08                          | Switch 1                         |
++--------------------------------+----------------------------------+
+| P0.09                          | Switch 2                         |
++--------------------------------+----------------------------------+
+| P0.06                          | Button 1                         |
++--------------------------------+----------------------------------+
+| P0.07                          | Button 2                         |
++--------------------------------+----------------------------------+
+| P0.17, P0.18, and P0.19        | Arduino pin headers              |
++--------------------------------+----------------------------------+
+| P0.21, P0.22, and P0.23        | Trace interface                  |
++--------------------------------+----------------------------------+
+| COEX0, COEX1, and COEX2        | COEX interface                   |
++--------------------------------+----------------------------------+
+
+For a complete list of all the routing options available,
+see the `nRF9160 DK board control section in the nRF9160 DK User Guide`_.
+
+If you want to route some of the above pins differently or enable any of the
+other available routing options, enable or disable the devicetree node that
+represents the analog switch that provides the giving routing.
```

Comment:
Did you mean "the given routing" or something like that?

Reply:
Yes. Corrected.